services/bettersocial/post/blockAnonymousPost.js

```diff
@@ -86,6 +86,7 @@ const BetterSocialBlockAnonymousPost = async (token, selfUserId, postId, source,
       message: e?.message || 'Error in block user v2 redis'
     };
   }
+  BetterSocialScoreBlockUser(selfUserId, authorAnonymousUserId, postId);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The invocation of `BetterSocialScoreBlockUser(selfUserId, authorAnonymousUserId, postId)` has been added. However, there is no error handling around this function call. If this function throws an error, it will stop the execution of the current function. Consider wrapping this function call in a try-catch block to handle potential errors.
```diff
+   BetterSocialScoreBlockUser(selfUserId, authorAnonymousUserId, postId);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->